sport/app/football/controllers/MatchController.scala

```diff
@@ -169,29 +175,38 @@ class MatchController(
 
   private def render(maybeMatch: Option[FootballMatch]): Action[AnyContent] =
     Action.async { implicit request =>
-      val response = maybeMatch map { theMatch =>
-        val lineup: Future[LineUp] = competitionsService.getLineup(theMatch)
-        val page: Future[MatchPage] = lineup map { MatchPage(theMatch, _) }
-        page map { page =>
-          if (request.forceDCR) {
+      maybeMatch match {
+        case Some(theMatch) =>
+          val lineup: Future[LineUp] = competitionsService.getLineup(theMatch)
+          val page: Future[MatchPage] = lineup.map(MatchPage(theMatch, _))
+          val tier = FootballSummaryPagePicker.getTier()
+
+          page.flatMap { page =>
             val footballMatch = NsAnswer.makeFromFootballMatch(theMatch, page.lineUp)
             val model = DotcomRenderingFootballMatchSummaryDataModel(
               page = page,
               footballMatch = footballMatch,
             )
```

Comment:
Just a suggestion, totally up to you if you think it's clearer this way.
Here we are making the `DotcomRenderingFootballMatchSummaryDataModel` model that's going to be used only if it's a DCR json or DCR renderer while the other cases won't need it. It might be more efficient to create the model separately for the 2 DCR cases in a similar way we did in here https://github.com/guardian/frontend/blob/main/sport/app/football/controllers/MatchListController.scala#L83